variants_colors/js/app.js

```diff
@@ -79,36 +72,32 @@ $(document).ready(function() {
     $hits.html(noResultsTemplate.render({query: content.query}));
   }
 
-
-
   // EVENTS BINDING
   // ==============
 
-  $searchInputIcon.on('click', function(e) {
+  $searchInputIcon.on('click', function (e) {
     e.preventDefault();
     $searchInput.val('').keyup().focus();
   });
-  $(document).on('click', '.clear-all', function(e) {
+  $(document).on('click', '.clear-all', function (e) {
     e.preventDefault();
     $searchInput.val('').focus();
     algoliaHelper.setQuery('').search();
   });
-  $(document).on('click', '.set-distinct', function(e) {
+  $(document).on('click', '.set-distinct', function (e) {
     e.preventDefault();
     $('#query-message').empty();
     setDistinct($(this));
     algoliaHelper.search();
   });
-  $(document).on('click', '.set-query', function(e) {
+  $(document).on('click', '.set-query', function (e) {
     e.preventDefault();
     $('#query-message').html($(this).data('message'));
-    setDistinct($(".set-distinct[data-value='"+$(this).data('distinct')+"']"));
+    setDistinct($(".set-distinct[data-value='" + $(this).data('distinct') + "']"));
```

Comment:
OH Wait, I know why it works, because you have single quotes inside it, nevermind